### ISSUE TITLE
Log created time is doubled.. Not needed in title of log

### ISSUE THREAD

**Issue by Pazekal90** [EVNotify/EVNotifyPWA#43]
![Bildschirmfoto 2019-07-17 um 22 35 02](https://user-images.githubusercontent.com/25208775/61409797-47f82c80-a8e3-11e9-80d8-12e376c0034a.png)

**Comment by GPlay97** [comment]
Yes, I see. The problem is, that this is the default Log name. Maybe it will change to a better name, best case would be location name (but the resolving is cost expensive). I will think about it.

**Comment by Pazekal90** [comment]
Already though about something lie this as the reason :) Maybe you can use an open source map api to get the location. openstreetmap for example.

**Comment by MattiJarvinen** [comment]
One option might be to use Nominatim (Open Streetmaps reverse geocode engine) for reverse geocoding. Not entirely cheap either, data volumes are huge and so are memory requirements too. https://nominatim.org/release-docs/latest/admin/Installation/

Anyways, Reverse geocoding with some GPS tolerance might allow some easy way to cache location reverse geocode data but done at scale the effect is just temporary.

Another option might be to allow the user to name a location themselves, that's not clean either.

Yet another option might be to use what 3 words, but does it really help at all? https://what3words.com/select-plan?currency=USD
